fleet_management_api/app.py

```
@@ -1,92 +1,159 @@
+    def _insert_key(self, uri: str) -> str:
+        if "?" in uri:
+            uri_base, query_params = uri.split("?")
+            return uri_base + f"?api_key={self._key}" + "&" + query_params
+        else:
+            return uri + f"?api_key={self._key}"
+
+    def _get_headers(self) -> dict[str, str]:
+        accessible_tenants = self._app._accessible_tenants
+        try:
+            token = get_token(*accessible_tenants)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)